src/api/controllers/getArticlesController.js

```diff
@@ -0,0 +1,63 @@
+import model from '../models/index';
```

Comment:
This controller should be renamed to ```articles``` @truestbyheart. Can you please do that?

Reply:
Already, done it